### DISCUSSION TITLE
Client side routing not working on new SSG site hosted on Vercel?

### DISCUSSION THREAD

**Discussion by chrisgreg** [vercel/next.js#19020] [Help] [answered]
I've just created [a new portfolio site](https://chriis-dev.vercel.app/) which seems to not be routing on the client side - seems like it's statically fetching and loading the new pages on navigation (try clicking on /work or the first blog post in the list).

I'm using getStaticProps in my index and blog pages but otherwise haven't done much of anything to disable features. 

Should client side routing be working or have I misinterpreted something?

**Comment by lfades** [comment, marked as answer]
I tried your site and it's working perfectly for me 👍 . If you still have the issue check your browser, make sure it's updated and try it on a page with extensions disabled. If it persists then please give us a reproduction we can take a look at 🙏 .

**Reply by chrisgreg** [reply]
Yep - I was just misusing the next/link with styled-components! 

Thanks!
